sdk/identity/identity/README.md

```diff
@@ -227,6 +227,26 @@ const credential = new ClientSecretCredential(
 );
 ```
 
+As an alternative to specifying the `authorityHost` argument, you can also set the `AZURE_AUTHORITY_HOST` environment variable to the URL of your cloud's authority. This approach is useful when configuring multiple credentials to authenticate to the same cloud:
+
+```sh
+AZURE_AUTHORITY_HOST=https://login.partner.microsoftonline.cn
+```
+
+The `AzureAuthorityHosts` enum defines authorities for well-known clouds for your convenience; however, if the authority for your cloud is not listed in `AzureAuthorityHosts`, you may pass any valid authority URL as a string argument as the following example demonstrates:
```

Comment:
In the `AzureAuthorityHosts` text above, link to https://learn.microsoft.com/javascript/api/@azure/identity/azureauthorityhosts?view=azure-node-latest. I think it's also confusing that the 1st occurrence calls it an interface, while the 2nd occurrence calls it an enum. Let's standardize on "enum".

Reply:
Updated, thanks!